README.rst

```diff
@@ -163,6 +163,8 @@ Dependencies
 * FPM_ (>= 1.0.1, although it might work with older versions too, only needed to
   build the Debian package)
 
+* Readchar_ (can be installed using ``pip install readchar``)
+
```

Comment:
Mmm, I don't like this, the idea was to have zero dependencies besides the Python stdlib, so you can download the script and run it without set up. Reading a char should be pretty easy, I don't think we need a library for that...